server/app.js

```diff
@@ -4,12 +4,13 @@ const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
 const mongoose = require("mongoose");
+const config = require("./config.js");
 
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
 
 // Connect to the database
-const mongoDB = "mongodb://127.0.0.1/mail-sender-dev";
+const mongoDB = config.mongoURI + ":" + config.mongoPort + "/" + config.mongoDB;
```

Comment:
nitpick: generally recommend template strings (``) vs. concatenation like this

Reply:
Makes sense. Fixed.